Sources/LiveViewNative/LiveView.swift

```diff
@@ -56,10 +56,26 @@ public struct LiveView<R: RootRegistry>: View {
                         {
                             ErrorView<R>(html: trace)
                         } else {
-                            SwiftUI.VStack {
-                                SwiftUI.Text("Connection Failed")
-                                    .font(.subheadline)
-                                SwiftUI.Text(error.localizedDescription)
+                            if #available(iOS 17, macOS 14, tvOS 17, watchOS 10, *) {
+                                #if swift(>=5.9)
+                                SwiftUI.ContentUnavailableView {
+                                    SwiftUI.Label("No Connection", systemImage: "network.slash")
```

Comment:
@carson-katri Are these Views able to be configured by the application developer so I can show a custom view when there's no connection, etc?

Reply:
Yes, use the `errorView` method on your `CustomRegistry` to provide a custom interface.